jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -50,20 +68,24 @@ def exit_cli(self) -> None:
         """
         self.app.exit(result=False)
 
-    def logout_exit_cli(self) -> None:
+    def logout_exit_cli(self, clear_all=False) -> None:
```

Ruff (0.15.10)

[warning] Boolean default positional argument in function definition

(FBT002)